index.html

```diff
@@ -0,0 +1,83 @@
+            <!--Status-->
+            <section class="status_TopNav">
+                <!--Hora-->
+                <div class="timeContainer">
+                    <a id="homeTime" href="javascript:void(0);">
+                        <img id="timeImage" src="images/Status/Time.png" alt="timeAncor">
+                    </a>
+                </div>
+                <!--Status-->
+                <div class="statusContainer">
+                    <a id="homeStatus" href="javascript:void(0);">
+                        <img id="statusImage" src="images/Status/Status.png" alt="statusAncor">
+                    </a>
+                </div>
+            </section>
+            <!--Nav bar-->
+            <section class="menuAngosto_TopNav">
+                <!--Logo-->
+                <div class="logoAngostoContainer">
+                    <a id="homeLogo_Angosto" class="homeLogo_Angosto">
+                        My Logo
+                    </a>
+                </div>
+                <!--Menu Hamburger-->
+                <div class="menuHamburgerContainer">
+                    <a id="menuHamburger" href="javascript:void(0);" class="icon">
+                        <img id="baraMenuImage" src="images/barMenu.png" alt="menuAncor">
+                    </a>
+                </div>
+            </section>
+        </nav>
+        <section>
+            <div class="mainPage">
+                <div class="flexContainer">
+                    <!--Container-->
+                    <div class="contentContainer">
+                        <h1>Hey There.<br/>I'm Alex.</h1>
+                        <h2>I'm a Software Developer</h2>
+                        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                            don’t hesitate to contact me.</p>
+                    </div>
+                    <!--Links a redes sociales Bottom-->
+                    <div class="socialLinks_Bottom">
+                        <a href="javascript:void(0);">
+                            <img id="gatoIconBot" src="images/Iconos/Gato.png" alt="iconGato">
+                        </a>
+                        <a href="javascript:void(0);">
+                            <img id="linkedinIconBot" src="images/Iconos/Linkedin.png" alt="iconLinkedin">
+                        </a>
+                        <a href="javascript:void(0);">
+                            <img id="manitoIconBot" src="images/Iconos/Manito.png" alt="iconManito">
+                        </a>
+                        <a href="javascript:void(0);">
+                            <img id="twitterIconBot" src="images/Iconos/Twitter.png" alt="iconTwitter">
+                        </a>
+                        <a href="javascript:void(0);">
+                            <img id="mIconBot" src="images/Iconos/M.png" alt="iconM">
+                        </a>
+                    </div>
```

Comment:
- [ ] Kindly use `ul` to group icons, see the below screenshot for further guidance
![Screenshot 2023-06-02 at 06 08 34](https://github.com/alexansaa/Personal_Portafolio/assets/112261237/d25d3cd4-abc3-42c7-953a-ca05ed83319f)
Kindly make the above changes